contrib/kubernetes/datahub/charts/datahub-frontend/templates/deployment.yaml

```diff
@@ -46,26 +46,38 @@ spec:
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           ports:
             - name: http
-              containerPort: 9001
+              containerPort: 9002
               protocol: TCP
           livenessProbe:
             httpGet:
               path: /admin
               port: http
+            initialDelaySeconds: 60
+            periodSeconds: 30
+            failureThreshold: 4
```

Comment:
I would probably make these come from `values`

Reply:
Should I make the whole readiness and liveness probe come from values?

Reply:
Done

Reply:
i have a suggestion here to only move below properties to values.yaml. The httpGet path can stay hardcoded as this the behavior of app which is not going to change on user wish.
```
  initialDelaySeconds: 60
  periodSeconds: 30
  failureThreshold: 8
```

Reply:
Done